Makefile

```diff
@@ -10,10 +10,7 @@ MANINSTALLDIR=${PREFIX}/share/man
 GO_MD2MAN ?= /usr/bin/go-md2man
 
 ifeq ($(DEBUG), 1)
-  GOGCFLAGS += "-N -l"
-else
-  GOGCFLAGS ?= ""
-  DEBUG ?= 0
+  override GOGCFLAGS += -N -l
```

Comment:
Moving the "" down and forcing override means GOGCFLAGS can't be set outside the make file. I didn't see a case where "GOGCFLAGS+=" was ignoring the debug=1 flag?